hangman.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'random-word'
+
+class Hangman
+  @word = ''
+  @guessedChars = []
+
+  def initialize
+    @word = RandomWord.nouns.next
```

Comment:
`Discussion`
How would we get a word from ANY source here, rather than relying on `Hangman` to know about `RandomWord` (or some other external client/gem?)